Frontend/CO.CDP.OrganisationApp/Program.cs

```diff
@@ -61,7 +61,14 @@
 
 builder.ConfigureForwardedHeaders();
 
-builder.Services.AddDistributedMemoryCache();
+builder.Services.AddStackExchangeRedisCache(options =>
+{
+    var redisHost = builder.Configuration.GetValue<string>("Aws:ElastiCache:Hostname");
+    var redisPort = builder.Configuration.GetValue<string>("Aws:ElastiCache:Port");
```

Comment:
Since this is part of Aws configuration, is it worth to make it explicitly part of the `AwsConfiguration` config class? 

https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/blob/main/Libraries/CO.CDP.AwsServices/AwsConfiguration.cs#L5-L14

With other Aws services we were also adding extension methods for convinnience, for example: 

https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/blob/6d5cff1853759835f03dc974a8910e85c4d07f5f/Frontend/CO.CDP.OrganisationApp/Program.cs#L215C1-L220C50